packages/openapi-typescript/test/schema-object.test.ts

```diff
@@ -374,48 +374,6 @@ describe("Schema Object", () => {
         expect(generated).toBe("0 | 1");
       });
 
-      test("empty object + oneOf is ignored", () => {
```

Comment:
We’ll want to keep this test—this is intended behavior.

Reply:
Oh, I didn't mean to delete this test at all! I'll fix it when I'm back at my laptop tomorrow.
Also, thanks for the kind words, much appreciated ☺

Reply:
Un-deleted the test now :)